python/sdk/tests/test_build.py

```diff
@@ -246,4 +246,58 @@ def test_update_detection(self, unwrap):
     def test_delete_detection(self, unwrap):
         unwrap(self.build.delete_detection)(self.build, detection_id=pytest.detection_id)
         with pytest.raises(Exception):
-            unwrap(self.detect.get_detection)(self.detect, detection_id=pytest.detection_id)
\ No newline at end of file
+            unwrap(self.detect.get_detection)(self.detect, detection_id=pytest.detection_id)
+
+@pytest.mark.order(4)
```

Comment:
intentional to use the same order index as the Detections tests? don't think the two should conflict, just  checking intention

Reply:
i have. but maybe that just means i'm a control freak. i'm fine with leaving this as is

Reply:
also, practically, when they have the same order index, i think they'll just run in order of the file. so detections will always run first anyway

Reply:
😆 makes no difference to me so I'm happy to increment

Reply:
if you do, you have to increment al the ones that come after it. and then probably also the order indexes on the delete test (last test of this lil module) as well.

Reply:
^ prefer you either leave the order index the same as the Detection tests or you update all the other ones in the other files too (test_detect, test_partner, test_probe, test_generate) so we're unique across the board